src/Microsoft.AspNet.Mvc.ViewFeatures/Rendering/ViewContext.cs

```diff
@@ -58,16 +58,6 @@ public ViewContext()
                 throw new ArgumentNullException(nameof(view));
             }
 
-            if (viewData == null)
```

Comment:
That's what I was originally doing, but @Eilon suggested we just leave the things null. He thought it should only become a problem if they tried to use it explicitly in their template, if there are calls using it which might NRE then I think that's a strong reason to just new-up an empty one, but on the other hand this also is a scenario you get to by returning a ViewResult directly, rather than letting us do all the plumbing, so I guess I can see it both ways. @Eilon, thoughts?